### PR TITLE
portfolio added

### PR DESCRIPTION
**Pull request by oerkperl**

### CODE REVIEW

File: app/components/portfolio/FormModal.tsx [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+      <Section>
+        <div className="flex  w-[1000px] h-[80vh] ">
+          <div className=" w-full flex items-center justify-center h-full border-gray-500  rounded-lg">
+            <div className="w-full border border-gray-400 p-8 rounded-xl">
+              <div className="flex justify-between">
+                <h1>Select coin</h1>
+                {hasError && <p>Network error: try again later</p>}
+                <button
+                  className="w-8 h-8  rounded-full bg-white dark:bg-gray-800 hover:bg-indigo-600 hover:text-white"
+                  onClick={() => onClose()}
```
Comment (yoloOnTheBattlefield) [inline app/components/portfolio/FormModal.tsx]: onClick={onClose}

File: app/components/portfolio/FormModal.tsx [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+                    className="h-12  w-full rounded-lg pl-2 bg-white dark:bg-gray-800"
+                    type="number"
+                    placeholder="Purchase amount"
+                    value={amount}
+                    onChange={handleAmountChange}
+                  />
+                  <div className="flex gap-2">
+                    <button
+                      className=" w-1/2 h-12 rounded-lg bg-pink-500 hover:bg-pink-600 text-white"
+                      onClick={() => onClose()}
```
Comment (yoloOnTheBattlefield) [inline app/components/portfolio/FormModal.tsx]: onClick={onClose}

and we usually call it handleClose

File: app/components/portfolio/FormModal.tsx [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+                    >
+                      Cancel
+                    </button>
+                    <button
+                      className=" w-1/2 h-12 rounded-lg bg-white dark:bg-gray-800 hover:bg-indigo-600 hover:text-white"
+                      disabled={!canAddAsset}
+                      onClick={() => {
+                        addNewAsset();
+                      }}
```
Comment (yoloOnTheBattlefield) [inline app/components/portfolio/FormModal.tsx]: onClick={addNewAsset}

File: app/components/portfolio/MyAsset.tsx [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+    <>
+      <div className="flex justify-between">
+        <h1>Market price</h1>
+
+        {hasError && (
+          <button
+            className="px-1 rounded-md hover:bg-indigo-600"
+            onClick={() => {
+              fetchAsset();
+            }}
```
Comment (yoloOnTheBattlefield) [inline app/components/portfolio/MyAsset.tsx]: onClick={fetchAsset}

File: app/components/portfolio/SelectCoin.tsx [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+          value={query}
+          onChange={handleChange}
+          placeholder="Search for a coin..."
+        />
+        {canSelect && (
+          <button
+            className="hover:text-indigo-500"
+            onClick={() => {
+              fetchData();
+            }}
```
Comment (yoloOnTheBattlefield) [inline app/components/portfolio/SelectCoin.tsx:86]: onClick={fetchData}

File: app/context/context.tsx [hunk 6]
```diff
@@ -30,13 +30,35 @@ const CryptoContextProvider: React.FC<{ children: ReactNode }> = ({
     { name: "btc", sym: "₿" },
     { name: "eth", sym: "Ξ" },
   ];
-
+  //const [assets, setAssets] = useLocalState<TAsset[]>("assets", []);
```
Comment (yoloOnTheBattlefield) [inline app/context/context.tsx]: no commented out code

**Review by yoloOnTheBattlefield — CHANGES_REQUESTED** [review]

File: app/context/context.tsx (hunk 6 above)
Comment (oerkperl) [reply app/context/context.tsx]: Ok to all.

**Pull request merged**